library/std/src/sys/thread_local/native/eager.rs

```diff
@@ -4,7 +4,7 @@ use crate::sys::thread_local::{abort_on_dtor_unwind, destructors};
 
 #[derive(Clone, Copy)]
 enum State {
-    Initial,
+    Uninitialized,
```

Comment:
`Uninitialized` isn't a great name, the value itself is very much initialised. It's just the destructor that's not registered yet.